content/src/service/synchronization/ContentCluster.ts

```diff
@@ -67,25 +67,7 @@ export class ContentCluster implements IdentityProvider {
 
         return { otherServers, lastSyncWithDAO: this.timeOfLastSync }
     }
-
-    getAddressForServerName(serverName: ServerName): ServerAddress | undefined {
```

Comment:
Could you delete the property `serverNames` and the method `getServerName` on this class?